readthedocs/starterguides/userguides/object_lifecycle_management_userguide.md

```diff
@@ -0,0 +1,89 @@
+# Opensds Object Lifecycle Management: User Guide
+
+## Introduction to Object lifecycle
+OpenSDS is aimed at addressing the storage integration challenges of both the Cloud Native environment and traditional IT environment. OpenSDS multi-cloud allows user to manage distributed cloud environment on a single platform. Object lifecycle management feature gives freedom to user to manage buckets and objects in the cloud using OpenSDS interface.
+
+## Getting Started!
+This guide is a tutorial which shows how to install, configure, and use Object lifecycle management feature in a simple OpenSDS setup.
+By following this guide you will be able to install OpenSDS multi-cloud with creating backends and buckets, uploading object and setting up the lifecycle for those objects.
+
+## Installing Opensds multi-cloud
+
+**Download opensds-installer code**
+
+    
+    git clone https://github.com/opensds/opensds-installer.git
+    cd opensds-installer
+    # Checkout the latest release, give master by default or specify the branch 
+    git checkout master
+   
+    
+  **How to enable multi-cloud installation**
+  
+ Install docker container to run multi-cloud in container environment.
+ 
+   Update the file : opensds-installer/ansible/group_vars/gelato.yml to install multi-cloud.
+   
+    # repository branch
+     gelato_repo_branch: master
+
+Detailed instruction is in this link
+
+- [OpenSDS Local Cluster Installation through Ansible On Ubuntu](https://github.com/opensds/opensds/wiki/OpenSDS-Cluster-Installation-through-Ansible) (Recommended)
+
+**export required OPENSDS  environment variables**
+```
+export HOST_IP={your_real_host_ip}
+export OPENSDS_ENDPOINT=http://{{ apiserver_cluster_ip }}:50040
+export OPENSDS_AUTH_STRATEGY=keystone
+export OS_AUTH_URL=http://{{ authchecker_cluster_ip }}/identity
+```
+**run OPENSDS multi-cloud in docker container**
+```
+   docker-compose up -d
+   docker ps 
+   ```
+   Note: check if multi-cloud services are running in the docker container.
```

Comment:
Insert output of `docker ps` with the containers that should be visible to the user.

Reply:
done